backend/src/main/kotlin/hu/bme/sch/cmsch/controller/admin/OneDeepEntityPage.kt

```diff
@@ -358,7 +358,7 @@ open class OneDeepEntityPage<T : IdentifiableEntity>(
     }
 
     @GetMapping("/show/{id}")
-    fun show(@PathVariable id: Int, model: Model, auth: Authentication): String {
+    open fun show(@PathVariable id: Int, model: Model, auth: Authentication): String {
```

Comment:
The idea here is to use the abstract template pattern. You could override variables and template functions in child classes but you could keep the original functions intact. I think you can achieve the same result without overriding and copying all the function to the child call. If you have a different logic you can copy/paste this and not inherit from One or Two deep entity page.